microsoft-edge/webdriver-chromium/index.md

```diff
@@ -61,279 +60,310 @@ To begin automating tests, make sure the WebDriver version you install matches y
+### Upgrading from Selenium 3
 
-If you're using Selenium 3, use the following steps to add the [Selenium Tools for Microsoft Edge](https://github.com/microsoft/edge-selenium-tools) and [Selenium 3](https://www.selenium.dev) to your project.
+The Microsoft Edge team recommends upgrading existing Selenium 3 tests to Selenium 4, because the Selenium project no longer maintains Selenium 3.  To learn more about upgrading to Selenium 4, navigate to [Upgrade to Selenium 4](https://www.selenium.dev/documentation/webdriver/getting_started/upgrade_to_selenium_4/).
 
-#### [C#](#tab/c-sharp/)
+If you're using [Selenium Tools for Microsoft Edge](https://github.com/microsoft/edge-selenium-tools) to add Microsoft Edge (Chromium) support to your Selenium 3 browser tests, update your tests as follows:
 
-Add the [Microsoft.Edge.SeleniumTools](https://www.nuget.org/packages/Microsoft.Edge.SeleniumTools) and [Selenium.WebDriver](https://www.nuget.org/packages/Selenium.WebDriver/3.141.0) packages to your .NET project using the [NuGet CLI](https://www.nuget.org/packages/NuGet.CommandLine/) or [Visual Studio](https://visualstudio.microsoft.com/).
+1. Remove [Selenium Tools for Microsoft Edge](https://github.com/microsoft/edge-selenium-tools) from your project.  It isn't necessary to use Selenium Tools for Microsoft Edge with Selenium 4, because Selenium 4 already has built-in support for Microsoft Edge.
 
-#### [Python](#tab/python/)
+1. Update your tests to use the built-in `EdgeDriver` and related classes that Selenium 4 provides instead.
 
-Use [pip](https://pypi.org/project/pip/) to install the [msedge-selenium-tools](https://pypi.org/project/msedge-selenium-tools/) and [selenium](https://pypi.org/project/selenium/) packages.
-
-```python
-pip install msedge-selenium-tools selenium==3.141
-```
-
-#### [Java](#tab/java/)
-
-If your Java project uses Maven, copy and paste the following dependency to your `pom.xml` file to add [msedge-selenium-tools-java](https://search.maven.org/artifact/com.microsoft.edge/msedge-selenium-tools-java/3.141.0/jar).
-
-```xml
-<dependency>
-    <groupId>com.microsoft.edge</groupId>
-    <artifactId>msedge-selenium-tools-java</artifactId>
-    <version>[3.141.0,)</version>
-</dependency>
-```
-
-The Java package is also available to download directly on the [Selenium Tools for Microsoft Edge Releases page](https://github.com/microsoft/edge-selenium-tools/releases).
-
-#### [JavaScript](#tab/javascript/)
-
-Use [npm](https://www.npmjs.com/) to install the [edge-selenium-tools](https://www.npmjs.com/package/@microsoft/edge-selenium-tools) and [selenium-webdriver](https://www.npmjs.com/package/selenium-webdriver) packages.
-
-```javascript
-npm install @microsoft/edge-selenium-tools selenium-webdriver
-```
+1. Remove all usages of the `EdgeOptions.UseChromium` property.  This property no longer exists in Selenium 4, because Selenium 4 supports Chromium-based Microsoft Edge only.
```

Comment:
```suggestion
1. Remove all usages of the `EdgeOptions.UseChromium` property.  This property no longer exists in Selenium 4, because Selenium 4 already supports Microsoft Edge (Chromium).
```

Reply:
Accepted but kept the word "only" because this should make it more clear that Chromium isn't just the default but is the only supported version.